tests/integration/api_v3/rpc/postTransactions.test.js

```diff
@@ -32,18 +32,20 @@ const {
 const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
 const postTransaction = async params => request(wsRpcUrl, 'post.transactions', params);
 
-// TODO: Enable test cases once given issue is resolved https://github.com/LiskHQ/lisk-sdk/issues/7172
-xdescribe('Method post.transactions', () => {
+describe('Method post.transactions', () => {
 	it('Post transaction succesfully', async () => {
 		const response = await postTransaction(
 			{
-				transaction: '0802100018002080c2d72f2a200fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a32250880c2d72f1214b49074a2eb04e7611908985f02c12fb7cd488d451a0874657374207478733a4065faed7b49d1ee63730cbb545ea25e50361581e35412eeffac3b35746afd1176d2ee1e270e8b072b1ccfad2d64f72918063c383003971600d56b168d6e429f05',
+				transaction: '0a05746f6b656e12087472616e7366657218032080c2d72f2a20a3f96c50d0446220ef2f98240898515cbba8155730679ca35326d98dcfb680f032270a0800000000000000001080c2d72f1a1474e3ba5ade3e94451bd4de9d19917c8e6eff624d22003a406049978f45e5664507a7a2a6a2fd0b76f98144fec26d54a0b73eac05c94cf23d4c794c25a82738c415640b28cddeeda1e9ea0d0a6e2268ab32666da2a49b3d06',
 			},
 		);
-		expect(response).toMap(postTransactionSchema);
+		const { result } = response;
+
+		expect(result).toBeInstanceOf(Object);
+		expect(result).toMap(postTransactionSchema);
 	});
 
-	it('invalid binary transaction -> empty response', async () => {
+	it.only('invalid binary transaction -> empty response', async () => {
```

Comment:
Same as above.
```suggestion
	it('invalid binary transaction -> empty response', async () => {
```

Reply:
WIP commit, kept a pointer for myself to fix.

Reply:
removed.